contrib/cr3bp_DhruvJ/cr3bp_char_quant.py

```diff
@@ -0,0 +1,156 @@
+2. To dimensioanlize a [nd] position vector: multiply by l*
+3. To non-dimensionalize a physical veloctiy vecotr, multiply by t*/l*
+4. To dimensionalize a [nd] veloctiy vecotr, multiply by l*/t*
+5. Position of P1 = [-mu, 0, 0] [nd] {x,y,z}
+5. Position of P2 = [1-mu, 0, 0] [nd] {x,y,z} 
+
+Physical Qunatities obtained from: JPL’s ephemerides file de405.spk and https://ssd.jpl.nasa.gov/?planet_pos
+"""
+import copy
+
+import numpy as np
+
+
+def mu_calc(mu_pi):
+    """Calculate mu of CR3BP
+    Parameters
+    ----------
+    mu_pi : ndarray, float
+        mu_pi[0] = mu of P1
+        mu_pi[1] = mu of P2
+
+    Returns
+    -------
+    mu: float, M2/(M1+M2)
+        M1 and M2 are mass of Primary Bodies and M2<M1
+    """
+    return mu_pi[1] / (mu_pi[0] + mu_pi[1])
+
+
+def tstar_calc(mu_pi, dist):
+    """Calculate t* of CR3BP
+    Parameters
+    ----------
+    dist: float, [nd]
+        Non-dimensional distance between P1 and P2
+    mu_pi: ndarray, float
+        mu_pi[0] = mu of P1
+        mu_pi[1] = mu of P2
+
+    Returns
+    -------
+    t*: float, [nd]
+        sqrt(dist^3/m*)
+        Non-dimensional time of P1-P2 system
+    """
+    return (dist**3 / (mu_pi[0] + mu_pi[1])) ** 0.5
+
+
+def bodies_char(b1, b2):
+    """Calculates mu, dist[km], t* [nd] of the 'body_i - body_j system'
+    Parameters
+    ----------
+    b1 : string
+       'body_i'
+    b2 : string
+       'body_j'
+
+    Returns
+    -------
+    mu: float
+        mu2/(mu1+mu2), mu2<mu1
+    dist: float, [nd]
+        Non-dimensional distance between P1 and P2
+    tstar: float, [nd]
+        sqrt(dist^3/m*)
+        Non-dimensional time of P1-P2 system
+    """
+    # Body values
+    mu = {}  # km^4 kg^-1 s^-2
+    mu["Sun"] = 132712440017.99
+    mu["Moon"] = 4902.8005821478
+    mu["Earth"] = 398600.4415
+
+    mu["Mars"] = 42828.314258067  # Mars, GM
+    mu["Jupiter"] = 126712767.8578  # Jupiter, GM
+    mu["Saturn"] = 37940626.061137  # Saturn, GM
+    mu["Uranus"] = 5794549.0070719  # Uranus, GM
+    mu["Neptune"] = 6836534.0638793  # Neptune, GM
+    mu["Pluto"] = 981.600887707  # Pluto, GM
+
+    mu["Phobos"] = 0.0007112  # Phobos, GM
+    mu["Titan"] = 8978.1382  # Titan, GM
+    mu["Ganymede"] = 9887.834  # Ganymede, GM
+    mu["Titania"] = 228.2  # Titania, GM
+    mu["Triton"] = 1427.598  # Triton, GM
+    mu["Charon"] = 102.30  # Charon, GM
+
+    #############
+    distances = {}  # km
+    distances["EarthMoon"] = 384400
+    distances["SunEarth"] = 149600000
+
+    distances["SunMars"] = 227944135
+    distances["SunJupiter"] = 778279959
+    distances["SunSaturn"] = 1427387908
+    distances["SunUranus"] = 2870480873
+    distances["SunNeptune"] = 4498337290
+    distances["SunPluto"] = 5907150229
+
+    distances["MarsPhobos"] = 9376
+    distances["JupiterGanymede"] = 1070400
+    distances["SaturnTitan"] = 1221865
+    distances["UranusTitania"] = 436300
+    distances["NeptuneTriton"] = 354759
+    distances["PlutoCharon"] = 17536
```

Comment:
I suppose these are mean distances?

Reply:
Yes. they are all the mean distances used for CR3BP models.

Reply:
Thanks, @JackCrusoe47! Yes, these are mean distances obtained from the JPL references stated at the top of the code. @astrojuanlu does poliastro have the mean distances saved anywhere? If not, do you think we should look into adding these constants to constants.py like file.